pkgs/applications/editors/emacs/elisp-packages/updater-emacs.nix

```diff
@@ -36,7 +36,6 @@ let
 in pkgs.mkShell {
   packages = [
     pkgs.git
-    pkgs.nix
```

Comment:
What about commenting this out and linking https://github.com/NixOS/nixpkgs/issues/349791 in the comment in case we want to revert this in the future?

Reply:
I don't see why we would want to add it again, and if we would we could revert the commit.
I prefer not to leave junk around.

Reply:
Pinning dependencies is good for reproducibility.  But yeah, generally, nix itself is not considered as a dependency.